tket/include/tket/Circuit/Circuit.hpp

```diff
@@ -762,8 +762,13 @@ class Circuit {
   // Basic Circuit Manipulation//
   //////////////////////////////
 
-  // O(V), `V` the number of vertices
-  void remove_blank_wires();
+  /**
+   * O(V), `V` the number of vertices
+   * removes all blank wires
+   * @param remove_classical_wire option to choose if empty classical wire
+   * should be removed as well
+   */
+  void remove_blank_wires(bool remove_classical_wire = true);
```

Comment:
I think it would be more intuitive to have a `keep_blank_classical_wires` argument that defaults to `false`.